### DISCUSSION TITLE
how to change localization?

### DISCUSSION THREAD

**Discussion by tyu543** [Sequel-Ace/Sequel-Ace#908] [General] [answered]
Hello,
I just wanna use English label.
how can I change localization to en?

Thanks.

**Comment by Jason-Morcos** [comment, marked as answer]
https://beebom.com/how-customize-language-settings-specific-apps-macos-catalina/
:)

**Reply by samuelbradshaw** [reply]
I didn't know that feature existed – nice!

**Reply by tyu543** [reply]
> https://beebom.com/how-customize-language-settings-specific-apps-macos-catalina/
> :)

It works. 
Thank you.
I don't know either.
^_^
